ansi2html/converter.py

```diff
@@ -325,7 +325,7 @@ def __init__(
             )
 
         self.vt100_box_codes_prog = re.compile("\033\\(([B0])")
-        self.ansi_codes_prog = re.compile("\033\\[" "([\\d;]*)" "([a-zA-z])")
+        self.ansi_codes_prog = re.compile("\033\\[" "([\\d;:]*)" "([a-zA-z])")
```

Comment:
Before more research I'm not sure this can get us into a false positives scenario or not.

Reply:
I confirm now that colon is working in xterm as advertized.